src/firefly/java/edu/caltech/ipac/astro/ibe/IBE.java

```diff
@@ -148,8 +148,8 @@ public URL createDataUrl(IbeDataParam param) throws IOException {
             qstr = addUrlParam(qstr, "gzip", param.isDoZip());
         }
 
-        //url += (qstr.length() > 0 ? "?" + qstr : "");
-        url += (qstr.length() > 0 ? "?" + URLEncoder.encode(qstr, "UTF-8") : "");
+        url += (qstr.length() > 0 ? "?" + qstr : "");
+        //url += (qstr.length() > 0 ? "?" + URLEncoder.encode(qstr, "UTF-8") : "");
```

Comment:
good

Reply:
Sorry, I don't mean to bud in, saw this went by my email.  
The bad commented line is already removed from `dev`.  You should remove it from here as well.  There's no reason to leave bad code in the source, even though it's commented out.

Reply:
good point. thanks.